kfdefs/overlays/osc/osc-cl1/jupyterhub/kfdef.yaml

```diff
@@ -29,17 +29,7 @@ spec:
           name: manifests
           path: radanalyticsio/spark/cluster
       name: radanalyticsio-spark-cluster
-    - kustomizeConfig:
-        parameters:
```

Comment:
any `kfdef` we're using `parameters` we should not remove them, but instead move them to the original `kustomizeConfig` for that component. So for instance add: 

```
    - kustomizeConfig:
        parameters:
          - name: s3_endpoint_url
            value: s3.odh.com
        repoRef:
          name: opf
          path: odh-manifests/osc/jupyterhub/jupyterhub
      name: jupyterhub
```

to the `kustomizeConfig` with `name: jupyterhub` and `path: jupyterhub/jupyterhub` (line 15)

same goes for the others

Reply:
think I got it, only jupyterhub and trino had params